doc/source/CHANGELOG.rst

```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented here.
 
 The format is largely inspired by keepachangelog_.
 
-.. _0.1.1:
+v0.7.1 - Unreleased
```

Comment:
I don't think this is a bugfix

Reply:
It isn't. But it's not going to be released on its own either. The final version will be assigned whenever something release-worthy gets pushed. This is more like a pre-emptive version bump.

Reply:
In fact, I think this is conflicting with master by now, and by now master is on 0.8.0, illustrating the point.